### PR TITLE
💄 32 - UI feedback

### PR DESCRIPTION
**Pull request by anncatton**
Addresses all the issues in #32 , except showing only configured providers, that will be handled in a separate ticket.

### CODE REVIEW

File: components/NavBar.tsx [hunk 1]
```diff
@@ -1,21 +1,31 @@
...
+    color: ${theme.colors.accent2_dark};
+  `;
+
   return (
     <div
       css={(theme: typeof defaultTheme) => css`
         display: flex;
         justify-content: space-between;
-        height: 50px;
+        height: ${NAVBAR_HEIGHT}px;
```
Comment (joneubank) [inline components/NavBar.tsx:28]: why isn't this a property in theme?

File: components/pages/explorer/PageContent.tsx [hunk 2]
```diff
@@ -1,24 +1,15 @@
...
-  `}
-`;
-
-const FACET_MAX_WIDTH = 270;
+import {
+  FACET_MAX_WIDTH,
+  FACET_MIN_WIDTH,
+  FOOTER_HEIGHT,
+  NAVBAR_HEIGHT,
+} from '../../../global/utils/constants';
```
Comment (joneubank) [inline components/pages/explorer/PageContent.tsx:12]: Is this an existing pattern or did you introduce this? These feel like values that usually get shared to CSS via the theme object.

File: global/utils/constants.ts [hunk 3]
```diff
@@ -4,3 +4,13 @@ const { NEXT_PUBLIC_EGO_API_ROOT } = getConfig();
...
+
+export const EXPLORER_PATH = '/explorer';
+export const USER_PATH = '/user';
+export const LOGIN_PATH = '/login';
+
+// dimension constants
+export const FACET_MAX_WIDTH = 270;
+export const FACET_MIN_WIDTH = 250;
+export const FOOTER_HEIGHT = 47;
+export const NAVBAR_HEIGHT = 50;
```
Comment (joneubank) [inline global/utils/constants.ts:16]: Repeating comment about css properties belonging in theme.

**Review by joneubank — CHANGES_REQUESTED** [review]

File: components/theme/dimensions.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
+// dimension constants
+export const FACET_MAX_WIDTH = 270;
+export const FACET_MIN_WIDTH = 250;
+export const FOOTER_HEIGHT = 47;
+export const NAVBAR_HEIGHT = 50;
```
Comment (joneubank) [inline components/theme/dimensions.ts:5]: we can get rid of these, ya?

**Review by joneubank — CHANGES_REQUESTED** [review]
Thanks for updates. You can get rid of the consts now.

File: components/theme/dimensions.ts (hunk 4 above)
Comment (anncatton) [reply components/theme/dimensions.ts:5]: oh whoops i thought i did. derp

**Review by joneubank — APPROVED** [review]
j'approve

**Pull request merged**